sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDto.java

```diff
@@ -62,27 +55,35 @@ public PermissionTemplateDto setName(String name) {
     return this;
   }
 
+  /**
+   * @deprecated since 5.2 use {@link #getUuid()}
+   */
+  @Deprecated
   public String getKee() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)